docs/releaseNotes.yml

```diff
@@ -32,6 +32,22 @@
 
 changelog: https://github.com/emissary-ingress/emissary/blob/$branch$/CHANGELOG.md
 items:
+  - version: 3.1.0
```

Comment:
In general, I'm thinking that we do not call this out here because it has no relevance to Emissary-ingress users.

@AliceProxy - thoughts on this?

Reply:
agreed